src/trunk.c

```diff
@@ -703,7 +703,11 @@ typedef enum trunk_compaction_type {
    NUM_TRUNK_COMPACTION_TYPES,
 } trunk_compaction_type;
 
-// arguments to a compact_bundle job
+/*
+ * Arguments to a compact_bundle job.
+ * Memory for the fingerprint array will be allocated, when needed,
+ * and hung off of breq_fingerprint.
+ */
```

Comment:
NOTE to reviewers: Fingerprint object handling and gyrations are very complex.

I had to struggle to get this right.

Kindly review this file very carefully.